tool/tctl/common/lock_command.go

```diff
@@ -76,6 +83,13 @@ func (c *LockCommand) CreateLock(ctx context.Context, client auth.ClientI) error
 	}
 	c.spec.Expires = lockExpiry
 
+	// If the user specify a server ID but not a node, copy the server ID to the node
+	// field. This is for backwards compatibility with previous versions of Teleport
+	// so that locking a node still works.
+	// TODO: DELETE IN 15.0.0
+	if c.spec.Target.ServerID != "" && c.spec.Target.Node == "" {
+		c.spec.Target.Node = c.spec.Target.ServerID
+	}
```

Comment:
Should this happen in the lock's `CheckAndSetDefaults` or in `(*auth.Server).UpsertLock` instead?

Reply:
Done in 7a6bfee379265e0e0eebb2abc1c28b91c897e033